### DISCUSSION TITLE
Error occurred: Cannot read properties of undefined (reading 'Create')

### DISCUSSION THREAD

**Discussion by soulfir** [pygame-web/pygbag#150] [Q&A]
Hi,
I've been getting the popup error "Error occurred: Cannot read properties of undefined (reading 'Create')" randomly when connecting to localhost:8000 in the browser. Sometimes it happens, sometimes it doesn't. When it does, I refresh the page a couple of times, and eventually, the error stops appearing and everything works. 
Any idea of what might be causing this and how to solve it?

**Comment by pmp-p** [comment]
your project is probably too big, what is currently the size of apk file you will find in build/web/ folder ?

**Reply by soulfir** [reply]
The apk file is currently 4.4MB. Is that too big?

**Reply by pmp-p** [reply]
no it is not, coud you share the whole project to see what is the problem ?

**Reply by psimaths** [reply]
I have the same problem but my apk file is 16.6 MB. Is that too big ?

**Reply by pmp-p** [reply]
would need: send apk, pygbag version used, os version used, was it on localhost and did it happened when using `--git` flag.

i cannot debug random errors without more insight.
